tests/e2e/cypress/integration/spanish.spec.js

```diff
@@ -0,0 +1,48 @@
+describe("Spanish language", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  })
+
+  it("Gives user Spanish language option", () => {
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish and does not change URL", () => {
+    cy.location("pathname").should("eq", "/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+    cy.location("pathname").should("eq", "/")
+  })
+
+  it("Changes the language from Spanish back to English", () => {
+    cy.contains("Español")
+      .click()
+
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+    cy.contains("English")
+      .click()
+
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish from any page", () => {
+    cy.get(".buttons .btn")
```

Comment:
Going off [this comment](https://github.com/cal-itp/benefits/pull/170#discussion_r746074038), I think we don't want to use the CSS selector? Or is there some reason we need to do it here?